api/api.yaml

```diff
@@ -111,6 +111,8 @@ definitions:
       search_placeholder_text:
         type: string
         description: Optional. What to show in the search box by default
+      enable_visualizations:
```

Comment:
`enable_visualizations` vs `show_visualizations` is a bit confusing. Let's rename `enable_visualzations` to `show_visualization_toggle`. Let's do this everywhere, including ui.json, API server, etc.